lib/ldap/model/ad/group.rb

```diff
@@ -28,6 +30,7 @@ class AD::Group < Base
       :display_name    => 'displayName',
       :account_name    => 'sAMAccountName',
       :members         => 'member',
+      :managed_by      => 'managedBy'
```

Comment:
Given that this attribute, if set, contains a distinguished name (DN), we can `.find()` and `return` a `LDAP::Model::AD::Person` rather than the LDAP DN string. WDYT?